content/guides/kubernetes/contour-to-ingress-and-beyond.md

```diff
@@ -0,0 +1,400 @@
+---
+title:  "Getting Started with Contour - To Ingress and Beyond"
+linkTitle: "Getting Started with Contour - To Ingress and Beyond"
+parent: Contour to Ingress and Beyond
+weight: 2
+topics:
+- Kubernetes
+- Tanzu
+tags:
+- Kubernetes
+- Tanzu
+- Contour
+- Ingress
+- httproxy
+- istio
+- service mesh
+- network
+# Author(s)
+team:
+- Paul Czarkowski
+---
+
+### Introduction to Contour
+
+Need to write some details about Contour here
+
+### Before You Begin
+
+You'll need a Kubernetes cluster. This guide uses a [Tanzu Kubernetes Grid](https://tanzu.vmware.com/kubernetes-grid) cluster, but any Kubernetes Cluster whether they're running on a Public Cloud, in your [Home] Lab, or on your desktop such as [KIND](https://kind.sigs.k8s.io/) or [minikube](https://minikube.sigs.k8s.io/docs/start/). You'll also need the Kubernetes CLI [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+
+1. Verify access to your Kubernetes cluster
+
+    ```bash
+    $ kubectl version --short
+    Client Version: v1.20.2
+    Server Version: v1.19.3+vmware.1
+    ```
+
+1. Create a scratch directory to work from
+
+    ```bash
+    mkdir ~/scratch/contour-demo
+    cd ~/scratch/contour-demo
+    ```
+
+### Installing Contour 1.12.0
+
+Since version 1.11.0 we've got two primary options for installing Contour, A singleton install from manifests, or by using the [Operator](https://projectcontour.io/resources/deprecation-policy/) (which is currently in Alpha). Since we only plan to install Contour once on the cluster, we can stick to the safer method of using the Contour provided manifests.
+
+You can install Contour directly from the manifests provided by the project, however best practice would have you download them locally first for validation and repeatability.
+
+1. Download contour installation manifests
+    ```bash
+    wget https://projectcontour.io/quickstart/contour.yaml
```

Comment:
Note that you can also do
```bash
wget https://projectcontour.io/quickstart/v1.12.0/contour.yaml
```
And get the `v1.12.0` version. That will work with any tagged release ( the base quickstart will get you the latest release).

Reply:
perfect! that was something I was going to figure out before publishing :)